proto/desmos/profiles/v1beta1/query_relationships.proto

```diff
@@ -29,15 +29,14 @@ message QueryUserRelationshipsRequest {
 // QueryUserRelationshipsResponse is the response type for the
 // Query/UserRelationships RPC method.
 message QueryUserRelationshipsResponse {
-  string user = 1;
 
   // relationships represent the list of all the relationships for the queried
   // user
```

Comment:
This can be removed now

Reply:
@RiccardoM How about adding a line to explain more about if the user is not provided, say:
```
// relationships represent the list of all the relationships for the queried user
// It represents all the relationships if user isn't provided
```

Reply:
I think it's not needed to specify it, it will be self explanatory once we rename the object itself